packages/plugins/src/api/index.js

```diff
@@ -25,6 +25,7 @@ const plugins = {};
  * @param {string|WPElement|Function} settings.icon   An icon to be shown in the UI. It can be a slug of the Dashicon,
  * or an element (or function returning an element) if you choose to render your own SVG.
  * @param {Function}                  settings.render A component containing the UI elements to be rendered.
+ * @param {number}                       settings.priority Allows for controlling the display order of this plugin. Default is 10.
```

Comment:
Nit, some whitespace issue:
```suggestion
 * @param {number}                    settings.priority Allows for controlling the display order of this plugin. Default is 10.
```

Reply:
It has been removed due to changes to the docBlock